titan/params/hiv.yml

```diff
@@ -31,6 +31,19 @@ hiv:
       type: float
       min: 0
       max: 1
+    haart_scale:
```

Comment:
since haart adherence is a bool, probably shouldn't make this a bin, but maybe an `adherent`/`non_adherent` pair like we do for `prep`

Reply:
hmmm, that would mean you can't easily access that key, I think?

Reply:
it would be the same as prep, with an `if`.  It's more verbose, but I think more idiomatic and less error prone (what if someone puts a bin 2?)